front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -119,6 +119,10 @@ const TopicsFilter = memo<Props>(
 
       const topicsWithIdeas = getTopicsWithIdeas(topics, filterCounts);
 
+      if (topicsWithIdeas.length < 1) {
```

Comment:
On second thought, we should probably still show this but with a "no topics available" message.